examples/platforms/nrf528xx/src/radio.c

```diff
@@ -943,13 +955,34 @@ void nrf_802154_receive_failed(nrf_802154_rx_error_t error)
     setPendingEvent(kPendingEventReceiveFailed);
 }
 
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
+uint16_t getCslPhase()
```

Comment:
style
```suggestion
uint16_t GetCslPhase(void)
```

Reply:
In this file, function names are like: `dataInit`, `convertShortAddress`, isPendingEventSet`, etc. Shall we keep the consistency here?

Reply:
fine with the name.
```suggestion
static uint16_t getCslPhase(void)
```